api/oidc.go

```diff
@@ -10,21 +10,24 @@ type OIDCToken struct {
 }
 
 type OIDCTokenRequest struct {
-	Job      string
-	Audience string
-	Lifetime int
-	Claims   []string
+	Job            string
+	Audience       string
+	Lifetime       int
+	Claims         []string
+	AwsSessionTags []string
```

Comment:
Minor [style nit](https://google.github.io/styleguide/go/decisions#initialisms):

```suggestion
	AWSSessionTags []string
```

Reply:
I'm clearly a Ruby developer first haha

Reply:
Popped on a commit which renames all mentions :+1: :pray: